controllers/horizon_controller.go

```diff
@@ -251,11 +249,12 @@ func (r *HorizonReconciler) SetupWithManager(mgr ctrl.Manager) error {
 }
 
 func (r *HorizonReconciler) reconcileDelete(ctx context.Context, instance *horizonv1beta1.Horizon, helper *helper.Helper) (ctrl.Result, error) {
-	r.Log.Info("Reconciling Service delete")
+	l := GetLog(ctx)
```

Comment:
Might be nice to keep a consistent name for this. Either `l` or `logger`. Makes it easier to find log messages with regex searching at least if the names are always consistent.

Reply:
This follows the existing naming in keystone-operator, and I think it makes sense because `l` and `logger` are initialized differently.